podcasts/SleepTimerManager.swift

```diff
@@ -39,6 +49,16 @@ class SleepTimerManager {
         }
     }
 
+    func restartSleepTimer() {
+        if let setting = Settings.sleepTimerLastSetting {
+            if let duration = setting.duration {
```

Comment:
We only restart when there's a timer going on with duration, as this wouldn't make much sense for the episode end.